files/build_templates/teamd.service.j2

```diff
@@ -5,9 +5,8 @@ After=updategraph.service
 
 [Service]
 User={{ sonicadmin_user }}
-ExecStartPre=/usr/bin/{{docker_container_name}}.sh start
```

Comment:
why do we remove this one?

Reply:
In stead of removing, we can just replace existing script.

---
In reply to: [232916218](https://github.com/Azure/sonic-buildimage/pull/2250#discussion_r232916218) [](ancestors = 232916218)

Reply:
i think it's better to align all these scripts later we could have some common functions that could simplify these scripts all together

Reply:
Then the unused scripts should be removed.